input/drivers/switch_input.c

```diff
@@ -88,12 +164,6 @@ static void switch_input_poll(void *data)
          sw->touch_previous_y[i] = sw->touch_y[i];
          sw->touch_x[i] = touch_position.px;
          sw->touch_y[i] = touch_position.py;
-         // prevent jumps in mouse pointer when putting down finger
```

Comment:
Comments need to be C-style (/* */), not C++-style.

Reply:
all fixed in latest force-push